src/components/landing/layout/footer/language-picker.tsx

```diff
@@ -11,12 +11,21 @@ const LanguagePicker: React.FC = () => {
     await i18n.changeLanguage(event.currentTarget.value)
   }
 
+  const shortenLanguageCode = ():string => {
+    const language = i18n.language
+    if (language.substr(0, 2) === 'zh') {
+      return language
+    } else {
+      return language.substr(0, 2)
+    }
+  }
+
```

Comment:
this function should have the type `(language: string): string`, it's not good style to get the language from i18n in the function...

Reply:
Could we get an enum for all available languages and return this enum as type? This would prevent invalid return values.

Reply:
I extracted the i18n reference. 

String as a return type is okay, because `i18n.language` returns a string. I can't verify that with an enum, because enums only exist in typescript. Not in the browser.